test/infrastructure/docker/.gitignore

```diff
@@ -1,20 +1,67 @@
-# invalid samples.
-config/samples
+# Binaries for programs and plugins
+*.exe
+*.dll
+*.so
+*.dylib
+cmd/clusterctl/clusterctl
+bin
+hack/tools/bin
+out
 
-# vscode
-.vscode
+# Test binary, build with `go test -c`
+*.test
+*.coverprofile.*
+*.coverprofile
 
-# goland
-.idea
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
 
-# ignore build output
-dist
-bin
+# IntelliJ
+.idea/
+*.iml
+
+# VSCode
+.vscode/
+
+# kubeconfigs
+minikube.kubeconfig
+
+# Book
+docs/book/book/
+
+# Common editor / temporary files
+*~
+*.tmp
+.DS_Store
+
+# rbac and manager config for example provider
+config/ci/rbac/role_binding.yaml
```

Comment:
might be I'm missing something, but why are we ignoring those files? if they are not necessary can we remove them from the project?

Reply:
I don't why as well. I think this is added by tilt. I didn't touch this file.